server/api/user/user.controller.js

```diff
@@ -257,10 +257,15 @@ exports.changeBio = function(req,res){
 
     User.findById(userId, function(err,user){
         user.bio = newBio;
+        if(newBio == ""){
```

Comment:
hmm I feel like this could get messy (because finding users by id in other places might have a different bio) again, @agundy